test/integrations/destinations/yandex_metrica_offline_events/processor/data.ts

```diff
@@ -0,0 +1,384 @@
+export const data = [
```

Comment:
Can we add test-cases for missing lines ?

Reply:
Added.